Makefile

```diff
@@ -112,6 +112,8 @@ format:
 	cd tests/test_runner && dune fmt || true
 	rustfmt $(RUSTFMT_FLAGS) $(INPUTS_DIR)/*.rs
 	cd $(INPUTS_DIR)/betree && cargo fmt $(RUSTFMT_FLAGS)
+	cd $(INPUTS_DIR)/naive-tree && cargo fmt $(RUSTFMT_FLAGS)
+	cd $(INPUTS_DIR)/avl && cargo fmt $(RUSTFMT_FLAGS)
```

Comment:
I think we should have one generic command to format all the subdirectories we find in `tests/src` (I can do that, I did it in other Makefiles)

Reply:
OK, I will leave those commits out then and leave it to you!